nav2_behavior_tree/README.md

```diff
@@ -75,5 +75,6 @@ The nav2_behavior_tree package provides several navigation-specific nodes that a
 | RandomCrawl | Action | This BT action invokes the RandomCrawl ROS2 action server, which is implemented by the nav2_experimental/nav2_rl/nav2_turtlebot3_rl experimental module. The RandomCrawl action server will direct the robot to randomly navigate its environment without hitting any obstacles. |
 | RecoveryNode | Control | The RecoveryNode is a control flow node with two children.  It returns SUCCESS if and only if the first child returns SUCCESS. The second child will be executed only if the first child returns FAILURE. If the second child SUCCEEDS, then the first child will be executed again. The user can specify how many times the recovery actions should be taken before returning FAILURE. In nav2, the RecoveryNode is included in Behavior Trees to implement recovery actions upon failures.
 | Spin | Action | Invokes the Spin ROS2 action server, which is implemented by the nav2_recoveries module. This action is using in nav2 Behavior Trees as a recovery behavior. |
+| PipelineSequence | Control | Ticks the first child till it succeeds, then ticks the first and second children till the second one succeeds. |
```

Comment:
Does this generalize to N nodes? If so it should say that. 

If it doesn’t, why doesn’t it? 

Also shouldn’t this instead retain state? Ei tick node A, then node B, then node... back to node A when things are working?

I’d say a FSMSequence would be a good name

Reply:
> Does this generalize to N nodes? If so it should say that. If it doesn’t, why doesn’t it?

Fair enough. I was trying to keep things simple in this doc, since I felt the other doc covered the full detail.

> Also shouldn’t this instead retain state? Ei tick node A, then node B, then node... back to node A when things are working?

I'm sure that makes sense for a node. It might be the `SequenceStar` node in the library. However, in this case, I want to tick the ComputePath and FollowPath subtrees on every top level tick.

ComputePath needs to be ticked to trigger replanning, and FollowPath needs to be reticked to update the plan and determine if Navigation has completed.